### ISSUE TITLE
Server packages

### ISSUE THREAD

**Issue by GPMueller** [GPMueller/arch-installer#50]
At this point we could start working on server packages. The installer is modular enough and we could try around with creating a small home media server.

**Comment by GPMueller** [comment]
`docker` should even be enough since services like nextcloud can be run over docker images.

**Comment by NOhs** [comment]
But do you have the latest versions of the software as docker containers like with native packages on Arch?

**Comment by GPMueller** [comment]
Probably not quite, but this may be an advantage in this case, as updating the system won't break the containers. Of course one then has to update the containers separately and ensure that migration works. But I suppose this may be easier with docker than doing it rather boilerplate by hand (especially migration).

I find it a bit hard to judge what the better approach is without experience, but I vote we try docker.

**Comment by NOhs** [comment]
For me this is ok if recent containers are available.  But I don't use a rolling release distro to install outdated software. Always having the latest packages is one of the major reason to use Arch for me.

**Comment by GPMueller** [comment]
I agree. From what I read it seems that one can automate the updating process, so that one could simply update docker images every time one updates the system. Probably they won't be quite as up to date, though...

When using it we will see if it's ok or not.

_FYI: if I understand correctly one can manually set up images, which could be given rolling updates, but I'm not sure how much work that would actually be. See this example: https://hub.docker.com/r/aheil/teamspeak3-server/_

**Comment by NOhs** [comment]
I could imagine that manual images are at least as much work as installing the latest arch packages, but we will see.

**Comment by GPMueller** [comment]
I verified with Markus C. that Docker is definitely the right way to go, using Nginx as reverse proxy for the docker containers one wants to expose. It would be advisable to setup everything using Dockerfiles.

We should also decide which server-specific services should go into autostart (`systemctl enable`). Created issue #72 for this.
